env.example

```diff
@@ -1,19 +1,17 @@
 # Replace the credentials here with your own.
 # Rename this file to .env before starting the app.
 
-# Watson Services
-# IMPORTANT: If your service is provisioned with IAM authentication, comment out the
-# username/password credentials and use the IAM credentials key/value pair
-
-# Watson Discovery authentication using username/password authentication
-DISCOVERY_USERNAME = <put discovery username here>
-DISCOVERY_PASSWORD = <put discovery password here>
-# Watson Discovery authentication using IAM
-#DISCOVERY_IAM_APIKEY=<put discovery IAM apikey here>
-#DISCOVERY_URL=<put discovery url here>
+# Replace the credentials here with your own.
```

Comment:
Same here.  Looks like these lines repeat (and my opinion on editing the original file).

Reply:
hmm... let me fix up. Not sure what happened.